src/components/GridTemplates/DetailsPage1/DetailsPage1.stories.js

```diff
@@ -0,0 +1,73 @@
+/**
+ * @file Details page template #1 story.
+ * @copyright IBM Security 2020
+ */
+
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+
+import { disableCentered, patterns } from '../../../../.storybook';
+import {
+  theme,
+  DetailsPage1,
+  DetailsPage1Main,
+  DetailsPage1Section,
+  DetailsPage1SectionTerm,
+  DetailsPage1SectionDefinition,
+  DetailsPage1Aside,
+} from '../../..';
+
+function DemoContent() {
+  return (
+    <div
+      style={{
+        backgroundColor: theme.interactive04,
+        minHeight: '240px',
+        width: '100%',
+      }}
+    >
+      lorem ipsum
```

Comment:
@cameroncalder should be able to provide some content that would be more descriptive for this example.

Reply:
We don't really have descriptive content anywhere in these components (just like "example" and "label"), but I can at least change these to just "content".

Reply:
You're right, we don't have much to work with right now! I'm envisioning a later story for these examples would be to leverage some of the components available to provide examples with additional context